package.json

```diff
@@ -32,6 +32,7 @@
     "chrome-launcher": "^0.15.0",
     "chrome-remote-interface": "^0.31.2",
     "lefthook": "^1.2.0",
+    "prettier": "2.7.1",
```

Comment:
I don't think this fits with the rest of the PR.

Reply:
ooops, I needed to install the latest prettier, must have auto-done this?